modules/dnn/test/test_tf_importer.cpp

```diff
@@ -146,6 +146,7 @@ TEST_P(Test_TensorFlow_layers, padding)
     runTensorFlowNet("padding_valid");
     runTensorFlowNet("spatial_padding");
     runTensorFlowNet("keras_pad_concat");
+    runTensorFlowNet("mirror_pad");
```

Comment:
The prefix should be similar to testing files: here it's `mirror_pad` but in https://github.com/opencv/opencv_extra/pull/622/files it's `mirrorpad`.

Reply:
Sorry, my mistake.

Reply:
@vonchenplus Please run test on your side with `--test_require_data=1 --gtest_filter=Test_TensorFlow_layers.padding/0` parameters